### DISCUSSION TITLE
Different tree format for scfl?

### DISCUSSION THREAD

**Discussion by simone-says** [iqtree/iqtree2#223] [Q&A]
I've been trying to calculate scfl for my dataset, but I keep getting an error 
```
Linked 19 total sequences
Reading input tree file /projects/tollis_lab/busco_phylo/squamates/results/06_astral/diplometopon_BUSCOs_WASTRAL_hybrid_species.tree ...ERROR: File not found or incorrect input, pls check it again. (line 2 column 0)
```
Even though I just used the same species tree to calculate gCF and sCF. These are my commands:
```
srun iqtree2 -t ${tree_dir}/diplo_BUSCOs_WASTRAL_hybrid_species.tree --gcf ${trees_dir}/diplo_BUSCOs_loci_iqtrees_cleaned.treefile -S ${aligns_dir} --scf 100 --df-tree --cf-verbose --prefix diplo_BUSCOs_WASTRAL_hybrid_CFs -st NT -T 20
srun iqtree2 -te ${tree_dir}/diplo_BUSCOs_WASTRAL_hybrid_species.tree -S ${aligns_dir} -m GTR --scfl 100 --cf-verbose --prefix diplo_BUSCOs_WASTRAL_hybrid_sCFL -st NT -T 20
```
Is there a different tree format required to run scfl??

**Comment by bqminh** [comment]
The error message "File not found or incorrect input" is informative. Please check again, if you're sure that the file exists, then pls send me the log file. 
This is a Q&A, so for further questions please post it in Discussions: https://github.com/iqtree/iqtree2/discussions.

**Comment by jasongallant** [comment]
Hi @bqminh and @simone-says , I'm having precisely the same issue for both v2.3.1 and v2.3.6.  In my case, the input tree is produced by astral4.  The same tree works fine for options --gcf and --scf, but I get the same error Simone reports when trying to run --scfl.  Pretty sure this is a bug, not a Q&A?  No information from the logs that is helpful in diagnosis.  Any advice would be greatly appreciated!

**Reply by jasongallant** [reply]
I have also checked and ASTRAL-III produced trees produce the same issue.

**Reply by simone-says** [reply]
My trees were generated using ASTRAL-V (i.e., WASTRAL) so I thought it may be something related to tree file format

**Reply by jasongallant** [reply]
Did you ever get it to work @simone-says ?

**Reply by simone-says** [reply]
Nope, just ran --scf like I did with older versions. I wanted to compared sCFs to the new max likelihood version but could never get it to work.

**Comment by roblanf** [comment]
@simone-says and @jasongallant are either of you able to post a reproducible example for this? At the moment it's a struggle for us to find time to get to the bottom of it, so anything you can do to help us get there would be massively appreciated.

**Comment by jasongallant** [comment]
Hi @roblanf is there somewhere I can send the data offline?

**Comment by roblanf** [comment]
Yeah for sure. You can email the files (if small) or a link (if big) to me: https://biology.anu.edu.au/people/academics/robert-lanfear

Most important are input files (as small as possible to reproduce the error), version number for IQ-TREE, OS, and command line.
